### DISCUSSION TITLE
Can releases be published to the "releases" URL of Tycho please?

### DISCUSSION THREAD

**Discussion by jonahgraham** [eclipse-tycho/tycho#1637] [Ideas]
At the moment if I visit https://github.com/eclipse-tycho/tycho/releases I am presented with a "There aren't any releases here" error:

![image](https://user-images.githubusercontent.com/679236/200448080-441032c0-259f-4405-bf24-8993352db9d1.png)

The release is announced in a message on discussions like this https://github.com/eclipse-tycho/tycho/discussions/1401#discussioncomment-3770187

Would it be possible to publish the releases on the standard GitHub URL and reference that from the discussion?

**Comment by laeubi** [comment]
@jonahgraham can you explain what particular advantage you see in publishing a explicit release record there? As Tycho is mostly release to maven central I'm not sure if we gain much with an additional "release" at github?


For example you can click on the tags:

https://github.com/eclipse-tycho/tycho/tags

if you want to download a specifics release source code and we have the Eclipse Relese records here:

https://projects.eclipse.org/projects/technology.tycho/governance

I also asked EF for having some kind of automation / synchronization between GH <-> EF but it has not getting much attention yet:

https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/1906

so if you have more insights or want recommend better practice it would be good to also share your ideas there,

All that said, I think there is nothing particular preventing us from doing so, maybe you like to contribute the steps you think are useful here:

https://github.com/eclipse-tycho/tycho/blob/master/RELEASING.md

so we can do this on the next release (and maybe for older ones as well).

**Comment by jonahgraham** [comment]
I created a PR in #1653

> @jonahgraham can you explain what particular advantage you see in publishing a explicit release record there? As Tycho is mostly release to maven central I'm not sure if we gain much with an additional "release" at github?

So that it appears in the "standard" place on GitHub. I originally was looked in https://github.com/eclipse-tycho/tycho/releases and didn't see details of 3.0.0 there, then (because I know Eclipse) I checked https://projects.eclipse.org/projects/technology.tycho/. Later I remembered you had told me that it was in GH discussions announcements too.

Note, [LSP4J](https://github.com/eclipse/lsp4j/) puts releases in the standard GitHub location too, even though most consumers consume it via maven or p2 (from download.eclipse.org).

We have discussed at architecture council about whether the GitHub releases can be an official place to have releases and not require them being entered in the PMI. I will [bring this back up at the next meeting](https://wiki.eclipse.org/Architecture_Council/Meetings/Meeting_Notes), along with your idea of using GitHub milestones as an official way of showing intended future releases.

**Reply by laeubi** [reply]
I have created a release for tycho 3.0 now. I would also not mind using this (maybe create a draft relase?) if the milestones do not suffice I just don't found a way to specify the intended release date there. So maybe it could/should be a combination of both.

**Reply by jonahgraham** [reply]
I think it should be a combination of both too.
